splunk-otel-android/src/main/java/com/splunk/rum/internal/opentelemetry/OpenTelemetryRum.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.rum.internal.opentelemetry;
+
+import io.opentelemetry.api.OpenTelemetry;
+
+/**
+ * Entrypoint for the OpenTelemetry Real User Monitoring library for Android.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
```

Comment:
This is temporary though, right? Like, eventually this will be a first-class interface that users interact with?

Reply:
Yep! I only added this comment now to make sure this does not become a part of our public API when we release 1.0.